### PR TITLE
Add convert to indexed

### PR DESCRIPTION
**Pull request by shayzluf**
[Part of] #2307 

---

# Add convert to indexed function v0.6

**Write why you are making the changes in this pull request**
to sync with the new spec

**Write a summary of the changes you are making**
added methods:
ConvertToIndexed
TestConvertToIndexed_OK

**Link anything that would be helpful or relevant to the reviewers**
https://github.com/ethereum/eth2.0-specs/blob/dev/specs/core/0_beacon-chain.md#convert_to_indexed

### CODE REVIEW

File: beacon-chain/core/blocks/block_operations.go [hunk 1]
```diff
@@ -541,6 +541,51 @@ func VerifyAttestation(beaconState *pb.BeaconState, att *pb.Attestation, verifyS
...
+//     custody_bit_1_indices = get_attesting_indices(state, attestation.data, attestation.custody_bitfield)
+//     custody_bit_0_indices = [index for index in attesting_indices if index not in custody_bit_1_indices]
+//     return IndexedAttestation(
+//         custody_bit_0_indices=custody_bit_0_indices,
+//         custody_bit_1_indices=custody_bit_1_indices,
+//         data=attestation.data,
+//         signature=attestation.signature,
+//     )
+func ConvertToIndexed(state *pb.BeaconState, attestation *pb.Attestation) (*pb.IndexedAttestation, error) {
+	attI, err := helpers.AttestationParticipants(state, attestation.Data, attestation.GetAggregationBitfield())
```
Comment (terencechain) [inline beacon-chain/core/blocks/block_operations.go]: why are you calling the getter? `attestation.GetAggregationBitfield()`

File: beacon-chain/core/blocks/block_operations.go [hunk 2]
```diff
@@ -541,6 +541,51 @@ func VerifyAttestation(beaconState *pb.BeaconState, att *pb.Attestation, verifyS
...
+	if err != nil {
+		return nil, err
+	}
+	cb1i, err := helpers.AttestationParticipants(state, attestation.Data, attestation.GetCustodyBitfield())
```
Comment (terencechain) [inline beacon-chain/core/blocks/block_operations.go]: same here

File: beacon-chain/core/blocks/block_operations.go (hunk 1 above)
Comment (shayzluf) [reply beacon-chain/core/blocks/block_operations.go]: changed

File: beacon-chain/core/blocks/block_operations.go (hunk 2 above)
Comment (shayzluf) [reply beacon-chain/core/blocks/block_operations.go]: changed

**Pull request merged**